python/app/widget_all_fields.py

```diff
@@ -85,7 +85,7 @@ def clear(self):
                 # set it's parent to None so that it is removed from the widget hierarchy
                 x.setParent(None)
                 # mark it to be deleted when event processing returns to the main loop
-                x.deleteLater()
+                x.destroy()
```

Comment:
Does this change contradict/is compatible with its own comment? Is there a reason it was done like this originally? (blame)

Reply:
As I wrote in the PR description, it seems `deleteLater` can cause conflicts with the gc. In my tests, this confirmed to fix the issue when opening the publish tab when it has items to display.

Reply:
Probably this line was implemented when Qt4 was still used (8y ago).